test/blackbox/common/DDSBlackboxTestsMonitorService.cpp

```diff
@@ -1230,8 +1393,10 @@ TEST_P(DDSMonitorServiceTest, monitor_service_simple_connection_list)
 
```

Comment:
In this case, no. The idea of the tests in the suite is that we are able to fill the `expected_msgs` beforehand, and one way of doing so, it is by defining the  ports in the XML, so that we can validate that the connections that came from the monitor service are the ones expected.